docs/assistant/use-attack-discovery-ai-assistant-incident-reporting.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[attack-discovery-ai-assistant-incident-reporting]]
+= Identify, investigate, and document threats
+
+:frontmatter-description: Elastic AI Assistant can help you write ES|QL queries.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [guide]
+:frontmatter-tags-user-goals: [get-started]
+
+Together, <<security-assistant, Elastic AI Assistant>> and <<attack-discovery,Attack discovery>> can help you identify and mitigate threats, investigate incidents, and generate incident reports in various languages so you can monitor and protect your environment. 
+
+In this guide, you'll learn how to:
+
+. <<use-case-incident-reporting-use-attack-discovery-to-identify-threats, Use Attack discovery to identify threats>>
+. <<use-case-incident-reporting-use-ai-assistant-to-analyze-a-threat, Use AI Assistant to analyze a threat>>
+. <<use-case-incident-reporting-create-a-case-using-ai-assistant,Create a case using AI Assistant>>
+. <<use-case-incident-reporting-translate,Translate incident information to a different human language using AI Assistant>>
+
+
+[discrete]
+[[use-case-incident-reporting-use-attack-discovery-to-identify-threats]]
+== Use Attack discovery to identify threats
+Attack discovery can detect a wide range of threats by finding relationships among alerts that may indicate a coordinated attack. This enables you to comprehend how threats move through and affect your systems. Attack discovery generates a detailed summary of each potential threat, which can serve as the basis for further analysis. Learn how to <<attack-discovery,get started with Attack discovery>>.
+
+image::images/attck-disc-11-alerts-disc.png[An Attack discovery card showing an attack with 11 related alerts,90%]
+
+In the example above, Attack discovery found connections between eleven alerts, and used them to identify and describe an attack chain.
+
+After Attack discovery outlines your threat landscape, use Elastic AI Assistant to quickly analyze a threat in detail. 
+
+[discrete]
+[[use-case-incident-reporting-use-ai-assistant-to-analyze-a-threat]]
+== Use AI Assistant to analyze a threat
+
+From a discovery on the Attack discovery page, click **View in AI Assistant** to start a chat that includes the discovery as context. 
```

Comment:
OK, now I'm seeing how maybe these sections DO go in a certain order, but I still think they could exist just as well as more standalone use cases. In this use case, we could suggest other UIs, workflows, etc. where a user can jump into AI Assistant to start analyzing — the Attack discovery page isn't the only jumping-off point.